validator/accounts/v2/new.go

```diff
@@ -200,14 +199,15 @@ func inputKeymanagerKind(_ *cli.Context) (v2keymanager.Kind, error) {
 func inputAccountPassword(_ *cli.Context) (string, error) {
 	var hasValidPassword bool
 	var walletPassword string
+	var err error
 	for !hasValidPassword {
 		prompt := promptui.Prompt{
 			Label:    "New account password",
 			Validate: validatePasswordInput,
 			Mask:     '*',
 		}
 
-		walletPassword, err := prompt.Run()
+		walletPassword, err = prompt.Run()
```

Comment:
This was previously shadowing and keeping the wallet password value within the scope of the loop, leading to a "" at the end of the function for a password value